lib/Installation/Partitioner/FormattingOptionsPage.pm

```diff
@@ -20,9 +20,7 @@ use parent 'Installation::WizardPage';
 
 use constant {
     FORMATTING_OPTIONS_PAGE => 'partition-format',
-    FILESYSTEM_SWAP         => 'partitioning_raid-swap_format-selected',
-    FILESYSTEM_FAT          => 'partitioning_raid-fat_format-selected',
-    FILESYSTEM_EXT4         => 'partitioning_raid-filesystem_ext4',
+    FILESYSTEM_TYPE         => 'partitioning_%s-format-selected',
```

Comment:
Hmm, I remember these changes from @sofiasyria PR for msdos test. Is it expected?

Reply:
I messed up rebasing :) fixed.